runtime/util/ConfigurationPrinter.cpp

```diff
@@ -284,3 +288,89 @@ void printSortedConfigurationToFile(
   writer w = {file, nullptr};
   printConfigurationInternal(&w, subject, sort, false, &state);
 }
+
+void printMatchResult(
+    std::ostream &os, MatchLog *matchLog, size_t logSize,
+    std::string &definitionPath) {
```

Comment:
Does this need to be a non-const reference?

Reply:
not at all! I'll change it to a const reference!